Test failing on OSX due to differences in temporary folder paths.

If I run "npm test" from a clean checkout on a Mac OSX 10.10.4 I get the following error:

 1) CLIEngine executeOnFiles() Fix Mode should return fixed text on multiple files when in fix mode:

      AssertionError: expected { Object (results, errorCount, ...) } to deeply equal { Object (results, errorCount, ...) }
      + expected - actual

         "errorCount": 1
         "results": [
           {
             "errorCount": 0
      -      "filePath": "/private/var/folders/_q/001g73y56vl24brvfcv0tl6m0000gn/T/eslint/fixtures/fixmode/ok.js"
      +      "filePath": "/var/folders/_q/001g73y56vl24brvfcv0tl6m0000gn/T/eslint/fixtures/fixmode/ok.js"
             "messages": []
             "warningCount": 0
           }
           {
             "errorCount": 0
      -      "filePath": "/private/var/folders/_q/001g73y56vl24brvfcv0tl6m0000gn/T/eslint/fixtures/fixmode/quotes-semi-eqeqeq.js"
      +      "filePath": "/var/folders/_q/001g73y56vl24brvfcv0tl6m0000gn/T/eslint/fixtures/fixmode/quotes-semi-eqeqeq.js"
             "messages": []
             "output": "var msg = \"hi\";\nif (msg === \"hi\") {\n\n}\n"
             "warningCount": 0
           }
           {
             "errorCount": 1
      -      "filePath": "/private/var/folders/_q/001g73y56vl24brvfcv0tl6m0000gn/T/eslint/fixtures/fixmode/quotes.js"
      +      "filePath": "/var/folders/_q/001g73y56vl24brvfcv0tl6m0000gn/T/eslint/fixtures/fixmode/quotes.js"
             "messages": [
               {
                 "column": 18
                 "line": 1

      at Function.assert.deepEqual (node_modules/chai/lib/chai/interface/assert.js:205:32)
      at Context.<anonymous> (tests/lib/cli-engine.js:700:24)

--- a/tests/lib/cli-engine.js
+++ b/tests/lib/cli-engine.js
@@ -731,20 +731,20 @@ describe("CLIEngine", function() {
                 assert.deepEqual(report, {
                     "results": [
                         {
-                            "filePath": path.resolve(fixtureDir, "fixmode/ok.js"),
+                            "filePath": fs.realpathSync(path.resolve(fixtureDir, "fixmode/ok.js")),
                             "messages": [],
                             "errorCount": 0,
                             "warningCount": 0
                         },
                         {
-                            "filePath": path.resolve(fixtureDir, "fixmode/quotes-semi-eqeqeq.js"),
+                            "filePath": fs.realpathSync(path.resolve(fixtureDir, "fixmode/quotes-semi-eqeqeq.js")),
                             "messages": [],
                             "errorCount": 0,
                             "warningCount": 0,
                             "output": "var msg = \"hi\";\nif (msg === \"hi\") {\n\n}\n"
                         },
                         {
-                            "filePath": path.resolve(fixtureDir, "fixmode/quotes.js"),
+                            "filePath": fs.realpathSync(path.resolve(fixtureDir, "fixmode/quotes.js")),
                             "messages": [
                                 {
                                     column: 18,

